app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
+  def update
+    @recipe = Recipe.find(params[:recipe_id])
+    @ingredient = Ingredient.find(params[:ingredient_id])
+    @recipe.ingredients << @ingredient
+    redirect_to "/recipes/#{@recipe.id}"
```

Comment:
This is minor, but a more RESTful way to build this controller action would be to consider a 3rd controller, the `RecipeIngredientsController` and use a `create` action there to indicate that you are creating a relationship between a recipe and an ingredient. 

For an application of this size, this is not a big deal, but imagine in a larger application, you might want to reserve the `recipes#update` to an action that just updates the attributes of a recipe (i.e. complexity, genre, etc).